src/Steps/Release/UpdateTranslations.php

```diff
@@ -124,6 +126,22 @@ protected function checkVersion(OutputInterface $output)
     }
 
     /**
+     * Test that yamlclean ruby gem is installed
+     *
+     * @param OutputInterface $output
+     * @throws InvalidArgumentException
+     */
+    protected function checkYamlCleanDependency(OutputInterface $output)
+    {
+        $error = "translate requires the yamlclean ruby gem. Run 'gem install yamlclean'";
+        $result = $this->runCommand($output, 'which yamlclean');
```

Comment:
Running `yamlclean` on its own returns the expected description, but is treated as an error so throws an exception... not sure why it’s treated as an error